src/wp-admin/authorize-application.php

```diff
@@ -132,11 +119,9 @@
 );
 
 require_once ABSPATH . 'wp-admin/admin-header.php';
-
 ?>
 <div class="wrap">
-	<h1><?php echo esc_html( $title ); ?></h1>
-
+	<h1><?php esc_html_e( 'Authorize Application' ); ?></h1>
```

Comment:
Since core strings are trusted, I think this would rather be:

```suggestion
	<h1><?php _e( 'Authorize Application' ); ?></h1>
```